frontend/ui/elements/menu_keyboard_layout.lua

```diff
@@ -8,13 +8,23 @@ local VerticalGroup = require("ui/widget/verticalgroup")
 local VirtualKeyboard = require("ui/widget/virtualkeyboard")
 local _ = require("gettext")
 
-local input_dialog, check_button_bold, check_button_border
+local input_dialog, check_button_bold, check_button_border, check_button_compact
 
 local sub_item_table = {
     {
         text = _("Keyboard layout"),
         sub_item_table = {},
     },
+    {
+        text = _("Remember last layout"),
```

Comment:
> Every new keyboard will be opened with "default" layout (except hide/show keyboard with a tap outside of input box).
Checkbox Remember last layout, default state: checked.

I don't understand, so does it remember the last layout or not?

Reply:
Sure, if checked (by default).

Reply:
If the input dialog is closed and the new one is opened, the layout is:
-checked: the same (last)
-unchecked: the default
If the keyboard is hidden/shown with a tap outside input box: the layout is always unchanged.